custom_components/tuya_local/devices/holman_wx8_irrigation_controller.yaml

```diff
@@ -196,8 +207,8 @@ secondary_entities:
         type: bitfield
         mapping:
           - dps_val: 64
-            value: true
-          - value: false
+            value: false
```

Comment:
Do all of the sensors need inverting?  Some of them may be obvious, such as Tuya flags batteries as OK=1, Low=0, while HA expects the flag to indicate the exceptional case Normal=0, Low=1, but the Evie sensor may have been correct according to what the other user reported.

Reply:
I think they are correct now. the Battery values I assume are all the same, so the Evie batter should be the same as the 9V battery state. The moisture and watering bits' show correct state changes when I trigger the system or manually trigger the rain sensor.